src/components/Tabs.react.js

```diff
@@ -124,53 +124,62 @@ export default class Tabs extends Component {
   }
   render() {
     let EnhancedTabs;
+    let selectedTab;
+    let selectedTabContent;
 
-    if (!Array.isArray(this.props.children)) {
-      // if dcc.Tabs.children contains just one single element, it gets passed as an object
-      // instead of an array - so we put in in array ourselves!
-      this.props.children = [this.props.children];
-    }
+    if(this.props.children) {
+      if (!Array.isArray(this.props.children)) {
```

Comment:
Some inconsistency here (`R.is(Array, foo)` is used above [here](https://github.com/plotly/dash-core-components/blob/70fe0d1cb885b4532b715a5265fad34cbe107ed5/src/components/Tabs.react.js#L37)). Apparently there's also the [undocumented](https://github.com/ramda/ramda/issues/249) `R.isArray`.